AppCreationScripts/sample.json

```diff
@@ -0,0 +1,92 @@
+{
+    "Sample": {
+      "Title": "An Angular single-page application that authenticates users with Azure AD and calls a protected ASP.NET Core web API",
+      "Level": 200,
+      "Client": "TodoListSPA",
+      "Service": "TodoListAPI",
+      "RepositoryUrl": "ms-identity-javascript-angular-spa-aspnetcore-webapi",
+      "Endpoint": "AAD v2.0"
+    },
+    "AADApps": [
+      {
+        "Id": "service",
+        "Name": "TodoListAPI",
+        "Kind": "WebApi",
+        "Audience": "AzureADandPersonalMicrosoftAccount",
```

Comment:
>AzureADandPersonalMicrosoftAccount [](start = 21, length = 34)

Updating this in the manifest throws the following error 
"Failed to update TodoListSPA application. Error detail: One or more of your reply urls is not valid. "

Reply:
I tried this now and it seems the `accessTokenAcceptedVersion` in manifest is set as null (which defaults to 1)... I suppose we should add another manual step to change that to 2 (but annoying that default is not 2...)